File: app/src/main/java/deltazero/amarok/receivers/DialerReceiver.java
package deltazero.amarok.receivers;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import deltazero.amarok.ui.MainActivity;

public class DialerReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        Intent launchIntent = new Intent(context, MainActivity.class);
        launchIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
        context.startActivity(launchIntent);
    }
} 

File: app/src/main/java/deltazero/amarok/receivers/DialerReceiver.kt
package deltazero.amarok.receivers

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import deltazero.amarok.ui.MainActivity

class DialerReceiver : BroadcastReceiver() {
  override fun onReceive(context: Context, intent: Intent) {
    val launchIntent = Intent(context, MainActivity::class.java)
    launchIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
    context.startActivity(launchIntent)
  }
}
